src/client/js/app-shell.js

```diff
@@ -1,3 +1,4 @@
+require('babel-polyfill')
```

Comment:
Actually it seems to make the bundle bigger, and I find it quite a deal breaker for something that should be smaller in principle ( fetch vs. axios ), I'll add a `.browserlistrc` to check if not supporting every browser on earth helps with that

Reply:
@temsa do you know how much bigger? The browserlist is definitely something we need to do - although probably warrants a separate ticket as it will require some investigation.

Reply:
It was ~90kb bigger. Finally replacing `babel-polyfill` for the browser by [transform-runtime](http://babeljs.io/docs/plugins/transform-runtime/) allowed to only keep what was necessary (thanks to tree shaking too). It is still not smaller than the original `axios` version , but at least it is only `8kb` bigger.
This is still frustrating and can probably be improved again, at least the tools are in place to do it (including a new `analyze` script to check the built bundle`